src/daemon/Language/Haskell/Tools/Daemon/Update.hs

```diff
@@ -1,12 +1,4 @@
-{-# OPTIONS_GHC -O0 #-} -- this is needed because of an optimization bug in GHC 8.2.1: https://ghc.haskell.org/trac/ghc/ticket/13413
```

Comment:
Probably backend issue.

Reply:
Nope, representation is OK, but you explicitely remove the pragmas with this assignment `filePragmas & annListElems .= newPragmas`.